boa_engine/src/module/mod.rs

```diff
@@ -233,6 +253,22 @@ impl ModuleLoader for SimpleModuleLoader {
 
         finish_load(result, context);
     }
+
+    // TODO: Try to unify `ModuleLoader::register_module` with `SimpleModuleLoader::insert`.
+    fn register_module(&self, _specifier: JsString, _module: Module) {
+        panic!(
+            "`SimpleModuleLoader` uses paths to cache the modules instead of specifiers.
+        To register a module, you need to use the `SimpleModuleLoader::insert` method."
+        )
+    }
+
+    // TODO: Try to unify `ModuleLoader::get_module` with `SimpleModuleLoader::get`.
+    fn get_module(&self, _specifier: JsString) -> Option<Module> {
+        panic!(
+            "`SimpleModuleLoader` uses paths to cache the modules instead of specifiers.
+        To get a module, you need to use the `SimpleModuleLoader::get` method."
+        )
+    }
```

Comment:
Should these really be panics or should it just be the default behaviour of doing nothing? Either way I think it probably should be documented on the `SimpleModuleLoader` struct so that users are informed right?

Reply:
Yeah, if we're going to add the functionality in the future, maybe this should just do nothing for now. Will add docs explaining this though.